angular-frontend/README.md

```diff
@@ -104,6 +104,48 @@ When developing local but would like to deploy the local code to the cluster the
 By using `Tanzu: Apply` on the `workload.yaml` it will create the Workload resource with the local source (pushed to an image registry) as
 starting point.
 
+## Authentication
+If you want to use the Tanzu AppSSO Authserver, there are a few things to consider
+
+### Inner loop (development mode)
+#### Configuring the `auth.conf.json`
+In your Angular app, there should be an `auth.conf.json`. The items in there refer to the following:
+```json
+{
+  "clientId": "your-client-id",
+  "authority": "https://your-authorization-server.com/",
+  "postLogoutRedirectUri": "http://localhost:4200/",
+  "redirectUri": "http://localhost:4200/",
+  "scope": [
+    "openid" // This list MUST contain openid
+  ]
+}
+```
+
+#### Configuring the Authserver
```

Comment:
**[suggestion]**: in this section, it is important to mention the `AuthServer.spec.cors` API and what the user needs to add for this accelerator to work.

Reply:
Thank you! I will add this.